src/components/text-area/TextAreaField.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+import * as React from 'react';
+import getProp from 'lodash/get';
+import type { FieldProps } from 'formik';
+
+import TextAreaPrimitive from './TextArea';
+import type { TextAreaProps } from './TextArea';
+
+type Props = TextAreaProps & FieldProps;
+
+const TextAreaField = ({ field, form, isRequired, ...rest }: Props) => {
+    const { name } = field;
+    const { errors, touched } = form;
```

Comment:
This means errors will show as a user tabs through a form, right? Do we want `touched` or some combination of `dirty`/`isSubmitted`?

Reply:
I can talk to design, but we can change this later too. If they are tabbing through and not filling out a required field, then showing error might be fine.